### PR TITLE
Hide CI files

### PR DESCRIPTION
**Pull request by beutlich**
.travis.yml needs to be in the root dir.

### CODE REVIEW

**Comment by beutlich** [comment]
@sjoelund Can you please check why

https://github.com/modelica/ModelicaStandardLibrary/blob/81ef5326885ef103e6f59e91e08e33cf628fc1f1/Modelica/Resources/BuildProjects/autotools/configure.ac#L17-L30

results in a [script error](https://test.openmodelica.org/jenkins/blue/organizations/jenkins/ModelicaAssociation%2FModelicaStandardLibrary/detail/PR-3329/3/pipeline#step-177-log-117) on Darwin? Thanks.

**Comment by sjoelund** [comment]
As for the OSX problem, that line should be `AC_MSG_RESULT([no])`; an error just exits the script...

**Comment by beutlich** [comment]
> 
> 
> As for the OSX problem, that line should be `AC_MSG_RESULT([no])`; an error just exits the script...

Figured it out by c52141529cafd7e39f358fe8b1f803c3f236b271 myself.

**Comment by sjoelund** [comment]
I installed the hook and the jobs are automatically in the queue. Just some jobs in front of it because someone triggered the nightly build during lunch :)

**Comment by beutlich** [comment]
Still failing ...

File: Modelica/Resources/BuildProjects/autotools/configure.ac [hunk 1]
```diff
@@ -12,8 +12,16 @@ AC_PROG_MAKE_SET
...
+AC_SUBST(LIBMATH)
 dnl Check for HDF5, etc
 
+LIBS_BEFORE="$LIBS"
+LIBS=""
+AC_SEARCH_LIBS([atan2], [m], [LIBMATH="$LIBS"], [
+  AC_MSG_ERROR([unable to find the atan2() function])
+])
+LIBS="$LIBS_BEFORE"
+
```
Comment (beutlich) [inline Modelica/Resources/BuildProjects/autotools/configure.ac]: There already is ``AC_SEARCH_LIBS([floor],[m])`.

**Comment by sjoelund** [comment]
Should be an easy fix to add the math to tables also. That test didn't fail on my local machine though...

**Pull request merged**